repl/repl.go

```diff
@@ -57,8 +61,121 @@ func Start(in io.Reader, out io.Writer) {
 		prompt.OptionPrefix("⧐  "),
 		prompt.OptionLivePrefix(changeLivePrefix),
 		prompt.OptionTitle("abs-repl"),
+		prompt.OptionHistory(getHistory()),
 	)
 	p.Run()
+	// we get here on ^D from the prompt
+	saveHistory()
```

Comment:
I would suggest to split out the history functionality in its own `repl/history.go` file just to keep things separate -- as I fear over time `repl.go` will become a beast :)

Reply:
Will do.